packages/subproviders/src/types.ts

```diff
@@ -11,7 +11,7 @@ export interface LedgerCommunicationClient {
  */
 export interface LedgerEthereumClient {
     getAddress_async: (derivationPath: string, askForDeviceConfirmation: boolean,
-                       shouldGetChainCode: boolean) => Promise<LedgerGetAddressResult>;
+                       shouldGetChainCode: true) => Promise<LedgerGetAddressResult>;
```

Comment:
keck?

Reply:
It's important. Other types assume that ledger returns chainCode so this helps to enforce that. So that the code can't pass false but still expect a chainCode.

Reply:
Hm... is this the right way to handle this? This should describe the interface to the `LedgerEthereumClient` and it can receive either value. I think before the code that relies on this being `true`, you should assert it and throw with a helpful error message if it is set to false. This way, it's explicit that the future coder needs to refactor the logic if they want to set it to `false`.

Reply:
Debatable. It's like defining the type for the whole module when you just use one function (extrapolating here). We only use one version of this function. So we restrict it with the type and the user will get a helpfull message. Also - where should we assert it? In our code?
```
const stuff = true;
assert(stuff);
call(smth, stuff);
```
Seems strange to me.

Reply:
I'd prefer type system to assertions system.